modules/RTC/TraceablePeerConnection.js

```diff
@@ -26,6 +26,8 @@ const SIM_LAYER_1_RID = '1';
 const SIM_LAYER_2_RID = '2';
 const SIM_LAYER_3_RID = '3';
 const SIM_LAYER_RIDS = [ SIM_LAYER_1_RID, SIM_LAYER_2_RID, SIM_LAYER_3_RID ];
+const SIM_LAYER_BITRATES = [ 500000, 700000, 2000000 ];
```

Comment:
Do we care to set a max for normal video encodings? If so maybe we should at least try to match the values in the client side engine https://cs.chromium.org/chromium/src/third_party/webrtc/media/engine/simulcast.cc?l=74&rcl=2f6e525099a0d5186e6b5b130aa26b78158d5aa7

Reply:
Thanks George, will take the max bitrate values from here.